doc/src/build/fullnode.md

```diff
@@ -1,58 +1,47 @@
 ---
-title: Run a Sui Fullnode
+title: Run a Sui Full node
 ---
 
-Sui full nodes run a service that
-stores the full blockchain state and history. They service reads, either for
-end clients or by helping other full nodes get up-to-date with the latest
-transactions that have been committed to the chain.
+Sui Full nodes run a service that stores the full blockchain state and history. They service reads, either for end clients or by helping other Full nodes get up-to-date with the latest transactions that have been committed to the chain.
```

Comment:
it's not accurate to say it stores the *full* blockchain data. Also it's worth calling out FN validates blockchain data.
`by helping other Full nodes get up-to-date with the latest transactions that have been committed to the chain.` this is WIP but not yet happening. 

```suggestion
Sui Full nodes run a service that validates blockchain activities, including transactions, checkpoints, and epoch changes etc. It stores and services the queries for the blockchain state and history.
```